commit f67fcbefce906cd419e4657f0d41e21019b71abd (HEAD -> formediapipe)
Author: Camillo Lugaresi <camillol@google.com>
Date:   Fri Aug 16 12:24:58 2019 -0700

    elementwise requires C++14

    This file fails to compile when using C++11, which is the default. This can be worked around by passing --cxxopt='-std=c++14' as a global build option to Bazel, but it is more convenient for users if we just configure this cc_library to be built with C++14 by default.

    The authors may also want to change it to be compatible with C++11, but that's out of scope for this change.

diff --git a/tensorflow/lite/delegates/gpu/metal/kernels/BUILD b/tensorflow/lite/delegates/gpu/metal/kernels/BUILD
index 17e59e70eb..4302a1f644 100644
--- a/tensorflow/lite/delegates/gpu/metal/kernels/BUILD
+++ b/tensorflow/lite/delegates/gpu/metal/kernels/BUILD
@@ -197,6 +197,9 @@ cc_library(
     name = "elementwise",
     srcs = ["elementwise.cc"],
     hdrs = ["elementwise.h"],
+    copts = [
+        "-std=c++14",
+    ],
     deps = [
         "//tensorflow/lite/delegates/gpu/common:model",
         "//tensorflow/lite/delegates/gpu/common:operations",
